products/gstreamer.md

```diff
@@ -0,0 +1,81 @@
+---
+title: GStreamer
+category: framework
+iconSlug: gstreamer
+permalink: /gstreamer
+versionCommand: gst-inspect-1.0 --version
+releasePolicyLink: https://gstreamer.freedesktop.org/
+changelogTemplate: https://gstreamer.freedesktop.org/data/pkg/src/1.11.90/{{"__RELEASE_CYCLE__"}}
+releaseLabel: "__RELEASE_CYCLE__"
+activeSupportColumn: false
+releaseDateColumn: true
+eolColumn: Supported
+
+auto:
+-   git: https://gitlab.freedesktop.org/gstreamer/gstreamer.git
+    regex: '^(?<major>[1-9]\d*)\.(?<minor>0|[1-9]\d*)\.?(?<patch>0|[1-9]\d*)?$'
```

Comment:
This regex must exclude odd minor releases because only the even minor release numbers look ready for production.